test/modules/MetricsValidator/src/Program.cs

```diff
@@ -37,10 +37,19 @@ static async Task<int> MainAsync()
                     .AddEnvironmentVariables()
                     .Build();
 
+                if (Environment.OSVersion.Platform != PlatformID.Unix)
+                {
+                    Logger.LogInformation("Windows detected, delaying");
+                    await Task.Delay(TimeSpan.FromMinutes(2));
```

Comment:
Are you going to figure out what is necessitating this delay before merging this PR?